docs/manual_api/cpp_tile_edsl.rst

```diff
@@ -0,0 +1,1070 @@
+C++ Tile eDSL
+#############
+
+The C++ Tile eDSL (Embedded Domain Specific Language) provides developers with a
+
+way of describing a neural network so that the Stripe-based PlaidML compiler can
+
+construct an efficient implementation.
+
+This tutorial is intended to help machine learning practitioners (or anyone with
+
+a background in software engineering and mathematics) get started using the C++
+
+Tile eDSL.
+
+Scope and Warning
+*******************
+
+This tutorial provides an introduction to the C++ Tile eDSL. It is intended to
+
+help machine learning practitioners get started writing Tile code as quickly as
+
+possible, and as such covers core features, not every language detail. This is a
+
+tutorial, not a spec, and as such will consist of a series of examples, with a
+
+summary reference section at the end.
+
+This tutorial covers how to use the C++ Tile eDSL, not how Tile code is
+
+constructed and manipulated by PlaidML. It does not cover the workings of
+
+PlaidML utilities such as the pmlc compiler.
+
+Tile and PlaidML are still being developed and the APIs discussed here are subject
+
+to change.
+
+How to Write Tile Code
+************************
+
+Sum Over Axis
+================
+
+We're ready to look at some C++ Tile code! Here's an operation that takes the
+
+sum over axis `0` of a 2D tensor (in Keras this would be ``K.sum(I, axis=0)``):
+
+.. code-block:: c++
```

Comment:
Use tabs to show c++ and python examples

Reply:
Done